doc/sphinxman/source/build_planning.rst

```diff
@@ -192,7 +192,9 @@ that software for |PSIfour| and any notes and warnings pertaining to it.
 
 * CMake (3.3+) http://www.cmake.org/download/
 
-* NumPy (needed at runtime, not buildtime) http://www.numpy.org/
+* NumPy (needed at runtime *and* buildtime) http://www.numpy.org/
```

Comment:
Why is numpy needed at build time, out of curiosity?

Reply:
Oh, I see now.  Nevermind!

Reply:
@dgasmith will have to answer, as I didn't trace the dep through. Basically comes back to generating the C code from python.

Only thing that scares me is if conda insists on embedding the numpy version dep into the p4 package. But surely I can turn that off, just haven't investigated yet.

Reply:
Gau2grid has a NumPy reference fallback version to use and test with.